styles/Caraousel.module.css

```diff
@@ -0,0 +1,35 @@
+.main {
```

Comment:
Nanti ini pake ccs in js aja ya, bisa pake styled-component by sass. nanti aku bikinin tutorialnya